### DISCUSSION TITLE
Sample for drawing an SVG pin?

### DISCUSSION THREAD

**Discussion by jameslavery** [Mapsui/Mapsui#2229] [Q&A] [answered]
I need to draw an SVG symbol/pin on the map, the intention being to display a circle with transparent fill.

Is there a sample of how to do this? I assume I create a Pin with an SVG type...

**Comment by charlenni** [comment, marked as answer]
See Samples/Mapsui.Samples.Common/Maps/Styles/SvgSample.cs

**Reply by jameslavery** [reply]
Thanks - should have found that myself!

**Reply by charlenni** [reply]
Correct 😉

Could you close this discussion?

**Comment by jameslavery** [comment]
Sure!

**Comment by pauldendulk** [comment]
You need to know that the Mapsui hit detection is based on color. So, if the center of the circle is fully transparent a tap inside it will not be detected as a hit. Sometimes that is what you want, so fine. If not, you could work around it by using a fill color which is almost fully transparent, like this:

```csharp
public static Color AlmostTransparent => new() { A = 1, R = 255, G = 255, B = 255 };
```

**Comment by jameslavery** [comment]
Thanks - that's definitely worth knowing!
________________________________
